test/javascripts/acceptance/new-topic-dropdown-test.js

```diff
@@ -0,0 +1,37 @@
+import { getOwner } from "@ember/owner";
+import { visit } from "@ember/test-helpers";
+import { test } from "qunit";
+import { acceptance, resetSite } from "discourse/tests/helpers/qunit-helpers";
+import selectKit from "discourse/tests/helpers/select-kit-helper";
+
+acceptance(
+  "DiscoursePresetTopicComposer - new topic dropdown",
+  function (needs) {
+    needs.user();
+
+    needs.hooks.beforeEach(() => {
+      resetSite({
+        topic_preset_buttons: [{ id: "NEW_BUG", name: "Report a bug" }],
+      });
+    });
```

Comment:
Is it possible to use `needs.site` here?

```suggestion
    needs.site({
      topic_preset_buttons: [{ id: "NEW_BUG", name: "Report a bug" }],
    });
```

Reply:
ah yes I over complicated things here thanks